scripts/core/registry.yml

```diff
@@ -592,6 +589,9 @@ etors:
 - name: ENQUEUE_NATIVE_COMMAND_EXP
   desc: Enumerator for $xEnqueueNativeCommandExp
   value: '228'
+- name: BINDLESS_IMAGES_RELEASE_EXTERNAL_MEMORY_EXP
```

Comment:
I am a little confused? The script can't detect renamed APIs so I removed the old one and introduced the new version of it with a sightly different name. Can you clarify what you mean by manually edit the function etors to retain the same value? Would that not cause problems with the script not being able to detect renamed APIs?

Reply:
In this case I think the entry point should have a different ID since the old name was alreayd existing on `main` so should have a different ID to remain globally unique.